### PR TITLE
add pf skip to gpu nfdhook

### PR DESCRIPTION
**Pull request by uniemimu**
Signed-off-by: Ukri Niemimuukko <ukri.niemimuukko@intel.com>

### CODE REVIEW

File: cmd/gpu_nfdhook/labeler.go [hunk 1]
```diff
@@ -87,6 +87,12 @@ func (l *labeler) scan() ([]string, error) {
 			continue
 		}
 
+		dat, err = ioutil.ReadFile(path.Join(l.sysfsDRMDir, f.Name(), "device/sriov_numvfs"))
```
Comment (mythi) [inline cmd/gpu_nfdhook/labeler.go:90]: just a thought but would it make sense to have a helper utility somewhere in `pkg/internal` that could be used by the plugins (the gpu plugin has the same check IIRC) to check if VFs are enabled.

Comment (eero-t) [reply cmd/gpu_nfdhook/labeler.go:90]: Commit message could say that this NFD labeler functionality corresponds to earlier GPU plugin change.

Potential name of the common helper function could be isSriovPFwithVFs().

(At some later point, the relevant copyright lines in the files could be updated to list also 2021.)

Comment (uniemimu) [reply cmd/gpu_nfdhook/labeler.go:90]: @mythi Not possible. "Internal packages (packages that are inside a folder that has an internal folder in their path) can only be imported from packages rooted at the parent of the internal folder."
I could perhaps introduce yet another "internal" folder under cmd, but given the makefiles, that might not work either.

Comment (uniemimu) [reply cmd/gpu_nfdhook/labeler.go:90]: Makefile needs this: "cmds = $(shell ls --ignore=internal cmd)"

**Review by bart0sh — APPROVED** [review]

**Pull request merged**